### PR TITLE
Test lower boundaries of version constraints

### PR DESCRIPTION
**Pull request by WouterJ**

### CODE REVIEW

**Comment by WouterJ** [comment]
@dbu do you know what's going wrong here? https://travis-ci.org/symfony-cmf/SimpleCmsBundle/jobs/64607348

The rendered html output: http://jsbin.com/tekelosoki/1/

**Comment by dbu** [comment]
this looks like we try to set the defaults property to an array
containing null values. this actually should work
https://github.com/jackalope/jackalope/issues/210 but that fix is not
yet finished. so i am not sure why this turns up - maybe phpcr-odm
meanwhile had a fix that skips null? or sonata admin?

to debug, i would try to run the tests locally and check if the latest
phpcr-odm still produces the problem, or sonata admin, or maybe one of
the other things in the stack. then go backwards until you find the
version that breaks and require something newer than that version.

**Comment by WouterJ** [comment]
Thanks for the information, @dbu. Using your debugging method, I found out it was fixed in `1.1.0-stable` (RC1 was installed). Now fingers crossed that this was the last problem...

**Comment by dbu** [comment]
yay! https://travis-ci.org/symfony-cmf/SimpleCmsBundle/jobs/64780699 is green!

File: .travis.yml [hunk 1]
```diff
@@ -21,22 +21,25 @@ matrix:
     - php: 5.6
       env: SYMFONY_VERSION=2.3.*
     - php: 5.6
-      env: SYMFONY_VERSION=2.5.*
-    - php: 5.6
       env: SYMFONY_VERSION=2.7.*
     - php: 5.6
       env: SYMFONY_VERSION=3.0.*
+    - php: 5.3.3
+      env: SYMFONY_VERSION="2.3.*,>=2.3.4" COMPOSER_FLAGS="--prefer-lowest"
```
Comment (dbu) [inline .travis.yml]: should the >=2.3.4 be in the composer.json? maybe in the form of conflicts: symfony < 2.3.4?

**Comment by lsmith77** [comment]
```Fatal error: Uncaught exception 'PHP_CodeCoverage_Exception' with message 'No code coverage driver available' in /home/travis/build/symfony-cmf/SimpleCmsBundle/vendor/phpunit/php-code-coverage/src/CodeCoverage.php:108```

I know that using a specific version of phpunit/php-codecoverage fixes the issue .. but I do not understand why the same issues doesn't affect CoreBundle/ContentBundle

**Comment by WouterJ** [comment]
@lsmith77 CoreBundle/ContentBundle use the PHPunit phar that is available on the travis machine (4.5.0). This bundle uses MatthiasSymfonyConfigTest, which requires phpunit. This means that executing `phpunit` will use the PHPunit classes that are available in the vendor directory (autoloading of vendor overrides autoloading of the phar), which is 5.0-dev.

**Comment by lsmith77** [comment]
ah

**Pull request merged**